setup.cfg

```diff
@@ -24,4 +24,6 @@ classifiers =
 python_requires = >=3.7
 
 [options.extras_require]
-tests = pytest
\ No newline at end of file
+tests = 
+    numpy
```

Comment:
numpy should be in `install_requires`.

Reply:
I'm personally OK removing Histogram.py.  I'm reasonably certain that no one is currently using it, and, as you say, if anyone is using it, they should change to something more modern.

Reply:
@HealthyPear, no, the pybind11 code also returns data as numpy arrays, which will fail if it is not installed.

Reply:
```
In [6]: f = xcdf.File('./test_file_for_pybindings.xcd', 'r')

In [7]: f
Out[7]: <xcdf.xcdf.File at 0x7fc8fe8a9830>

In [8]: next(f)
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Cell In [8], line 1
----> 1 next(f)

ModuleNotFoundError: No module named 'numpy'
```

Reply:
right. - fixed